models/Group.ts

```diff
@@ -153,32 +165,45 @@ const updateExistingRecordingsForGroupWithMatchingStationsFromDate = async (
       stations
     );
     if (matchingStation !== null) {
-      recordingOpPromises.push(new Promise(async (resolve) => {
-        await recording.setStation(matchingStation);
-        resolve({
-          station: matchingStation,
-          recording
+      recordingOpPromises.push(
+        new Promise(async (resolve) => {
+          await recording.setStation(matchingStation);
+          resolve({
+            station: matchingStation,
+            recording
+          });
         })
-      }));
+      );
     }
   }
   return recordingOpPromises;
 };
 
 export interface Group extends Sequelize.Model, ModelCommon<Group> {
   id: GroupId;
+  groupname: string;
   addUser: (userToAdd: User, through: any) => Promise<void>;
   addStation: (stationToAdd: CreateStationData) => Promise<void>;
-  getUsers: (options?: { where?: any, attributes?: string[] }) => Promise<User[]>;
-  getDevices: (options?: { where?: any, attributes?: string[] }) => Promise<Device[]>;
+  getUsers: (options?: {
+    through?: any;
```

Comment:
through?

Reply:
can be used to filter on the through table e.g. `through: { where: { admin: true } }`